facts-styles.css

```diff
@@ -0,0 +1,47 @@
+ul .nav-parent {
+    background-color: lightblue;
+    grid-row: 2 / 5;
+    grid-column: 1;
+
+}
+
+h3,
+h4 {
+    text-align: center;
+}
+
+.column {
+    float: left;
+    width: 55%%;
+}
+
+  /* Clear floats after the columns */
+.row:after {
+    content: "";
+    display: table;
+    clear: both;
+}
+img {
+    width: 150px;
+    height: 200px;
+    display: block;
+}
+.Chewie,
```

Comment:
Technically valid to name classes capitalized but you'll find most classes are all lowercase with kebab-case, e.g. `.example-cls`